lib/validator.ts

```diff
@@ -0,0 +1,47 @@
+import { INestApplication } from '@nestjs/common';
+import { SwaggerSchemes } from './swagger-schemes';
+import { ValidatorErrorHandler } from './validator-error-handler';
+import Ajv from 'ajv';
+
+const ajv = new Ajv({
+  strict: false,
+  removeAdditional: true,
+  useDefaults: true,
+  coerceTypes: true,
+  allErrors: true,
+});
+
+export class Validator {
```

Comment:
Создай папку в `lib` и назови её `validators`. А этот валидатор переименуй в конкретный

```suggestion
export class AjvValidator {
```

Reply:
+